examples/pxScene2d/src/Makefile

```diff
@@ -153,23 +176,40 @@ LDNODEV8 = -L$(NODEDIR)/out/Release
 #LDNODE   = -L$(NODEDIR)/out/Debug/obj.target
 #LDNODEV8 = -L$(NODEDIR)/out/Debug
 
+ifeq ($(EMBEDDED_BUILD),1)
 LDWESTEROS = -L$(EXTDIR)/westeros/external/install/lib -lwesteros_compositor
+else
+LDWESTEROS = $(EXTDIR)/westeros/external/install/lib/libwayland-server.so -L$(EXTDIR)/westeros/external/install/lib -lwesteros_compositor
+endif
 
 
 LDEXT   = $(LDPNG) $(LDJPG) $(LDFT) $(LDCURL) $(LDNODE) $(LDNODEV8) $(LDZLIB)
 ifeq ($(HAVE_WESTEROS),1)
 LDEXT += $(LDWESTEROS)
 endif
 ifeq ($(UNAME_S),Linux)
+ifeq ($(USE_X11),1)
 LDEXT += -lX11
 endif
+endif
 
 ifeq ($(UNAME_S),Darwin)
 LDFLAGS = -L$(PXLIBS) -lnode -lpxCore -pthread $(LDEXT) -ldl -lv8_libplatform -framework OpenGL -framework Cocoa -framework Foundation
 else
-LDFLAGS = -L$(PXLIBS) -lnode -lpxCore -pthread -lglut -lGLEW -lGL $(LDEXT) -ldl -lrt -lv8_libplatform
+ifeq ($(PX_PLATFORM),PX_PLATFORM_WAYLAND_EGL)
+LDFLAGS = -L$(PXLIBS) $(PXCORE_LIBS) -lnode -lpxCore -pthread $(LDEXT) -lwayland-egl -lEGL  -lwayland-client -ldl -lrt
+ifeq ($(EMBEDDED_BUILD),1)
+LDFLAGS += -lGLESv2
+else
+LDFLAGS += -lGL
+endif
+else
+LDFLAGS = -L$(PXCORE_LIBS) -L$(PXLIBS) -lnode -lpxCore -pthread -lglut -lGLEW -lGL $(LDEXT) -ldl -lrt -lv8_libplatform
+endif
+ifeq ($(RTRPCENABLED),1)
```

Comment:
Where did RTRPCENABLED come from and why is it needed?  Please remove and refer to previous comments.